examples/generator-fullstack/model/datamodel.graphql

```diff
@@ -2,14 +2,14 @@ type Note {
   id: ID!
   title: String!
   description: String!
-  ## Relationship
-  comment: [Comment!]!
 }
 
 type Comment {
   id: ID!
   title: String!
   description: String!
+  # Relationship
+  note: Note!
```

Comment:
Defining relationship from this side instead makes it easier to represent in mutation. Not a long term solution though.

Reply:
Actually we have it done by bidirectional relationships - they are added to each side if missing. We also have a single unified way to represent OneToOne and OneToMany so it works both ways on the server. The only problem is the resolver for other direction is not defined and client-side do not support stuff at all. This is something I was trying to quickly patch. 
To summarize - relationships are fundamentaly broken because:
- Legacy migration always builds oneToMany relationship
- Schema generator does bidirectional relationships (adding fields that were not defined in model)
- resolvers for birectional relationships are missing
- client side generator is missing the relationships queries

Ways to fix it:

- Agree on how we represent relationships comprehensively 
- Build a reliable model to represent relationships in `InputTypeModelContext` object.